src/PlutusTx/LedgerTypes.hs

```diff
@@ -0,0 +1,127 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module PlutusTx.LedgerTypes where
+
```

Comment:
nit; consider placing this in Language.PureScript.Bridge.Plutus module if it makes sense to you